homeassistant/components/sonos/__init__.py

```diff
@@ -92,17 +107,19 @@ def _stop_discovery(event):
             data.hosts_heartbeat()
             data.hosts_heartbeat = None
 
-    def _discovery(now=None):
+    def _discovery(now: datetime.datetime | None = None) -> None:
         """Discover players from network or configuration."""
         hosts = config.get(CONF_HOSTS)
 
-        def _discovered_player(soco):
+        def _discovered_player(soco: SoCo) -> None:
             """Handle a (re)discovered player."""
             try:
+                _LOGGER.debug("Reached _discovered_player, soco=%s", soco)
 
                 data = hass.data[DATA_SONOS]
 
                 if soco.uid not in data.discovered:
+                    _LOGGER.debug("Adding new entity")
```

Comment:
Probably unneeded debug message

Reply:
This has been useful a couple of times while moving things around. Shouldn't hurt to keep it for now.